CONTRIBUTING-students.md

```diff
@@ -56,6 +74,37 @@ a mentor for the project
 and you can start with [our mentor list][ML].
 **Proposals without a mentor will not be considered.**
 
+### How to write a great Proposal
+
+Firstly, think about your choice of project carefully, you're going to be doing it for a couple of months, so it's important that you choose something you're going to enjoy. Once you've made your mind up:
+
+1. Make sure you've thought about the project and understand what it entails
+2. Contact us early! The earlier you contact us the earlier you will be able to get feedback from us to improve your application
+3. Don't be afraid to come up with original solutions to the problem
+4. Don't be afraid to give us lots of detail about how you would approach the project
+
+Overall, your application should make us believe that you are capable of completing the project and delivering the functionality to our users. If you aren't sure about anything, get in touch with us, we're happy to advise you.
+
+### Dividing your project
+
+We require that all of our students have at least one commit in the development branch of your project before the end of the summer. The best way to achieve this is to divide your project into small self contained subprojects and plan to merge at least one of them around midterm. 
```

Comment:
What about rewrite as

> We require that all of our students create at least one **new** commit reviewed by their mentors before each term. The best way to achieve this is to divide your project into small self contained subprojects. If you don't send any commit for your mentor review each term we will fail you but don't worry about it because we are sure that you will create dozens if not hundreds of commits for your mentors review over the summer.